tests/test_convert_lit_checkpoint.py

```diff
@@ -101,3 +101,47 @@ def test_against_original_gpt_neox():
     ours_y = ours_model(x)
     theirs_y = theirs_model(x)["logits"]
     torch.testing.assert_close(ours_y, theirs_y)
+
+
+@torch.inference_mode()
+@pytest.mark.parametrize("size", ("7b", "70b"))
+def test_against_llama2(size):
```

Comment:
For consistency with the other test names
```suggestion
def test_against_original_llama2(size):
```